src/CommunityToolkit.Maui.Core/Views/SemanticOrderView/MauiSemanticOrderView.android.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Linq;
+using Android.Content;
+using Android.Views;
+using Microsoft.Maui.Platform;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The native implementation of the SemanticOrderView control.
```

Comment:
```suggestion
/// The native implementation of the <see href="SemanticOrderView"/> control.
```

Reply:
Side note, I just copied these from the platform popup control so the MauiPopup platform view probably also needs these fixes